### ISSUE TITLE
Stylistic whitespace breaks plural format

### ISSUE THREAD

**Issue by jkronborg** [jeffijoe/messageformat.net#27]
Unit test:
```diff
diff --git a/src/Jeffijoe.MessageFormat.Tests/MessageFormatterFullIntegrationTests.cs b/src/Jeffijoe.MessageFormat.Tests/MessageFormatterFullIntegrationTests.cs
index 5148af6..29af464 100644
--- a/src/Jeffijoe.MessageFormat.Tests/MessageFormatterFullIntegrationTests.cs
+++ b/src/Jeffijoe.MessageFormat.Tests/MessageFormatterFullIntegrationTests.cs
@@ -334,6 +334,13 @@ namespace Jeffijoe.MessageFormat.Tests
                         new Dictionary<string, object?> { { "count", 3 } },
                         "You and 2 others added this to their profiles."
                     };
+                yield return
+                    new object[]
+                    {
+                        "{ count, plural, one {1 thing} other {# things} }",
+                        new Dictionary<string, object?> { { "count", 2 } },
+                        "2 things"
+                    };
             }
         }
```

Result:
```
Failed Jeffijoe.MessageFormat.Tests.MessageFormatterFullIntegrationTests.FormatMessage(source: "{ count, plural, one {1 thing} other {# things} }", args: [[count, 2]], expected: "2 things") [1 ms]
  Error Message:
   Assert.Equal() Failure
           ↓ (pos 1)
Expected: 2 things
Actual:   2
           ↑ (pos 1)
```

Would be nice if whitespace after the opening brace was allowed 🙂 Other MessageFormat implementations that we are using support it, and I would like to not care about/enforce a specific style.

**Comment by jeffijoe** [comment]
That makes sense!

I'm a bit swamped with work right now but I'll get to it.

**Comment by jeffijoe** [comment]
Fixed in 6.0.0 😄
